lib/team_api/joiner.rb

```diff
@@ -101,37 +108,22 @@ def initialize(site)
       @public_mode = site.config['public']
     end
 
+    # Jekyll seems to be removing non-alpha characters from the team member
+    # names, causing the TeamIndexer to not be able to find any team member
+    # matches. This changes all of the team member keys back to what we expect.
+    def restructure_team_data!
+      if !site.data['team'].nil? && site.data['team'].respond_to?(:keys)
```

Comment:
`.present?` is a method available with `ActiveSupport` (http://stackoverflow.com/a/18810587) - Rails isn't a dependency in this project so we are without the added niceties it provides.